specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_azure_cognitive_search_options.tsp

```diff
@@ -23,15 +24,20 @@ model AzureCognitiveSearchIndexFieldMappingOptions {
 
   @doc("The names of index fields that should be treated as content.")
   @projectedName("csharp", "ContentFieldNames")
-  contentFieldNames?: string[];
+  contentFields?: string[];
```

Comment:
Should this model be renamed to `AzureCognitiveSearchIndexFieldsMappingOptions` (notice "Fields" instead of "Field") for consistency?

Reply:
I think it's right with "field mapping options." Just like you could eat multiple hot dogs while following "hot dog eating instructions," the infix is about the rules/patterns irrespective of how many instances they're applied over.